README.md

```diff
@@ -105,8 +105,25 @@ Yamashita | Yah-MAH-sh-Ta  | some vowels (i, u) get removed (called devoicing),
 Kyoko     | Kyoh-koh       | (not Kee-oh-koh)      
 Ryousuke  | Ryoh-skeh      | the -suke suffix is usually skeh
 
+## Portuguese (Brazilian)
+
+The challenge with Portuguese is that there are some phonems (such as \[ɐ̃w̃\], \[s̻\] or \[ʎ\]) and [diacritics](https://en.wikipedia.org/wiki/Diacritic) (such as an accent or cedilla) not common or not at all present in the English language. 
+
+First, a few general rules:
+1) **ão** is a nasal diphthong and resembles "pawn" + "o"
```

Comment:
love the description... most people don't know what a nasal diphthong is though... so maybe just say something like a sound that resonates in your nose and moves from the vowel in "pawn" to "oh" in a smooth glide?  (assuming i'm interpreting that right... haven't studied pt-br)

Reply:
I think a better way to describe might be to tell people, “Say the English word ‘pawn’ but say the vowel like you’re congested”.

At least for me that worked pretty well.

Reply:
If any of you would like to try it out on a call, happy to help.  I know this is a really tricky one.